sites/dotnet-guide/contents/tutorials/aspnet-basics/configure-services/index.md

```diff
@@ -0,0 +1,32 @@
+---
+type: TutorialStep
+date: 2020-05-22
+title: Dependency Injection and ConfigureServices
+technologies: [asp.net]
+products: [rider]
+topics: [web]
+author: khalidabuhakmeh
+subtitle: Registering ASP.NET services and instances
+thumbnail: ./thumbnail.png
+longVideo: 
+    poster: ./poster.png
+    url: https://www.youtube.com/watch?v=a32WK_wyM2E&feature=youtu.be
+---
+
+We'll be focusing on the `ConfigureServices` method and performing registration through the `IServiceCollection` interface.
+
+Throughout this video, We'll see the behavior of our `HelloWorldService` change. We will inject the `HelloWorldService` into a Razor page, and the changes will become clear as we switch between the registration types in `ConfigureServices`.
+
+We'll first register our dependency as `Transient`. The creation of our `HelloWorldService` will occur every time our application asks for an instance. Changing our registration to Scoped will create a single instance for each new web request. Finally, registering a dependency as a `Singleton` will limit our `HelloWorldService` to a single instance for the duration of our application's run.
+
+By changing the registration of our dependencies, we can dramatically alter the behavior in our ASP.NET application.
+
+
+### See Also
+
+- [Microsoft Docs Startup](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/startup)
+- [Dependency Injection](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection)
+- [JetBrains Rider](https://jetbrains.com/rider)
```

Comment:
For tutorial steps, I'd consider not linking Rider/JetBrains generic pages (you're already on our web property). Instead maybe link one or two relevant blog posts or web help articles?

Linking to generic ".NET", seems too verbose, too, maybe.